spread/client.go

```diff
@@ -468,13 +476,21 @@ func (c *Client) runPart(script string, dir string, env *Environment, mode outpu
 	}
 
 	if e, ok := err.(*ssh.ExitError); ok && e.ExitStatus() == 213 {
+		lines := bytes.Split(bytes.TrimSpace(stdout.Bytes()), []byte{'\n'})
+		m := commandExp.FindSubmatch(lines[len(lines)-1])
+		if len(m) > 0 && string(m[1]) == "ERROR" {
+			return nil, fmt.Errorf("%s", m[2])
+		}
+	}
+
+	if e, ok := err.(*ssh.ExitError); ok && e.ExitStatus() == 214 {
```

Comment:
I think this needs a comment why we now have two different exit codes and why the old detection no longer works. It's also slightly dangerous as now any exit code 214 in -vv mode will trigger a reboot. I wonder if there is a way to make this nicer. I will meditate a bit.

Reply:
I'll need to reproduce the error because I don't remember exactly the details

Reply:
I already removed the new exit code and left 213 where the -vv is handled